docs/modules/record-encryption/aws-kms/con-aws-kms-setup.adoc

```diff
@@ -12,17 +12,18 @@ filter to be kept separate from any keys used by other systems. This document as
 will be to prefix the alias used by the filter with the word `KEK_`. If a different naming convention is used, adapt
 the instructions accordingly.
 
-== Administrator Actor
+== Role of the Administrator
 
-To use the filter, there must be an administrative identity established within AWS IAM.  This user, which is likely to be a human,
-has the responsibility to manage keys and aliases within AWS KMS for use by the filter.
+In order to use the filter, there must be an *Administrator*.  The Administrator has the responsibility
+to create encryption keys within AWS KMS which will be used to encrypt the records. It is up to the organization
+deploying Record Encryption to decide how the role of the *Administrator* maps into their organization structure.
 
-Use AWS IAM to create a `kroxylicious-admin` user. Attach the policies `AWSKeyManagementServicePowerUser` and `IAMUserChangePassword`
-to the user. You may wish to attach policy `AWSCloudShellFullAccess` so the Administrator can use the AWS CloudShell to
-use CLI to manage the KEKs. Grant the user access to the Console.
+The *Administrator* must have permissions to log in to AWS and create keys in AWS KMS.  As a starting point, the
```

Comment:
I'd say `log in to AWS` is implicit within `create keys in AWS KMS` without pre supposing a human interaction.